src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -110,6 +114,7 @@ public ClaimService(
         this.paidInFullRule = paidInFullRule;
         this.ccdEventProducer = ccdEventProducer;
         this.claimAuthorisationRule = claimAuthorisationRule;
+        this.asyncEventOperationEnabled = Boolean.getBoolean(asyncEventOperationEnabled);
```

Comment:
You can use `boolean` type in parameter, This conversion of `Boolean.getBoolean(..)` will not be needed.

Reply:
is removed